xml/deployment_prep_zseries_prep_i.xml

```diff
@@ -246,40 +234,40 @@
      After you IPLed from the SCSI DVD as described in
      <xref
       linkend="sec-info-iplopt-dvd"/>, the installation system uses
-     the DVD as the installation medium. In that case, you do not need the
+     the DVD as the installation medium. In this case, you do not need the
      installation media on an FTP, NFS, or SMB server. However, you need the
      network configuration data for your &productname;, because you must set up
-     the network during the installation to perform a graphical installation by
+     the network during the installation to perform a graphical installation via
      VNC or by X.
     </para>
    </sect4>
   </sect3>
   <sect3 xml:id="sec-prep-makeavail-zpxe-cobbler">
    <title>Using a Cobbler Server for zPXE</title>
    <para>
-    IPLing from the network requires a Cobbler server, to provide the kernel,
+    IPLing from the network requires a Cobbler server to provide the kernel,
     initrd, and the installation data. Preparing the Cobbler server requires
-    four steps:
+    the following steps:
    </para>
    <itemizedlist>
     <listitem>
      <para>
-      Importing the Installation Data
```

Comment:
Here, I would prefer the '-ing' form because it is in sync with the headlines of the sections below and makes it easier for the reader to identify the matching sections that follow. In fact, you could even insert xrefs to the respective sections in this overview, so anyone who has already performed some of the steps, can directly jump to 'their' next step (if you insert ```xrefstyle="select:label nopage"``` you can omit the page number for the link and only get the title of the section you are pointing to) .